build/tools/cloudcore.service

```diff
@@ -4,6 +4,8 @@ Description=cloudcore.service
 [Service]
 Type=simple
 ExecStart=/etc/kubeedge/cloudcore
+Restart=always
```

Comment:
Is that meaning service will always try when the exception cannot get recovered?

Reply:
This "always" means that the service will be restarted regardless of whether the exit code is 0 or not.

Reply:
I don't think it's a problem. And kubelet dose the same.

> [Unit]
> Description=kubelet: The Kubernetes Node Agent
> Documentation=https://kubernetes.io/docs/
> 
> [Service]
> ExecStart=/usr/bin/kubelet
> Restart=always
> StartLimitInterval=0
> RestartSec=10
> KillMode=process
> 
> [Install]
> WantedBy=multi-user.target

Reply:
I'm fine with this